src/Monomer/Widgets/Animation/Fade.hs

```diff
@@ -12,12 +12,9 @@ Messages:
 
 - Accepts an 'AnimationMsg', used to control the state of the animation.
 -}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE Strict #-}
```

Comment:
@Deltaspace0 I'd be wary of removing this one. I know `Strict` affects the programming style and, in most cases, can be avoided by careful use of bang patterns, but at least we should have `StrictData` enabled.

Reply:
Ok, I will put it back, I just removed it blindly along with other extensions since all the logic was moved to Transform module and many extensions were no longer needed. I didn't look why the `Strict` extension was there in the first place, though I should have.

Reply:
@Deltaspace0 no worries!